tool/tbot/utils/destination.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package utils
```

Comment:
- can you think of a better name than `utils`? It's a slippery slope and ends up becoming a bag of misc code.
- Add a newline between the license and the package clause so that `go doc` doesn't interpret the license as documentation.

Reply:
Fair point, I'll just rename it to `destination` - just needs to be its own package to prevent import cycles.

Those load-bearing newlines are fun!